Scripts/fastlane/Fastfile

```diff
@@ -282,6 +267,35 @@ import "./ScreenshotFastfile"
     sh("cd .. && rm WordPress.app.dSYM.zip")
   end
 
+  ########################################################################
+  # Fastlane match code signing
+  ########################################################################  
+  private_lane :internal_code_signing do |options|
+    match(
+      type: "enterprise",
+      team_id: ENV["INT_EXPORT_TEAM_ID"],
```

Comment:
I wonder – is there a way to read this out of the project configuration (or even hardcode it?) 

I had set up the new environment variable on my machine, which wasn't a huge deal, but wondering if we can shave a minute or two off of the setup this way.

Reply:
I had the same thought, but just went with what was already here. @loremattei what do you think? Is there any reason not to hardcode the team id here?